packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/stop-bot-modal.tsx

```diff
@@ -7,37 +7,48 @@ import { isMobile } from '@deriv/shared';
 type TStopBotModal = {
     is_running: boolean;
     is_dialog_open: boolean;
+    is_contract_dialog_open: boolean;
+    is_stop_bot_dialog_open: boolean;
     onOkButtonClick: () => void;
     toggleStopBotDialog: () => void;
+    stopMyBot: () => void;
+    closeMultiplierContract: () => void;
+    is_multiplier: boolean;
```

Comment:
@farabi-deriv it would be better to keep them in the order we usually keep: first the variables, then the functions

Reply:
Sure. Done ✅